cfgov/paying_for_college/jinja2/paying-for-college/disclosure_technote.html

```diff
@@ -409,9 +404,7 @@ <h4>
                             period
                         </li>
                         <li class="list_item">
-                            Loan fee of <span data-financial="DLOriginationFee"
-                            data-fee="origination"
-                            data-percentage_value="true">[XX]</span>% of the
+                            Loan fee of {{ DLOriginationFee }}   of the
```

Comment:
Could get rid of the extra space after the variable, or I can get it in my PR.

Reply:
```suggestion
                            Loan fee of {{ DLOriginationFee }} of the
```